pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -45,10 +45,26 @@ type PerconaXtraDBClusterSpec struct {
 }
 
 type PXCSpec struct {
-	AutoRecovery *bool `json:"autoRecovery,omitempty"`
+	AutoRecovery        *bool                `json:"autoRecovery,omitempty"`
+	ReplicationChannels []ReplicationChannel `json:"replicationChannels,omitempty"`
+	Expose              ServiceExpose        `json:"expose,omitempty"`
 	*PodSpec
 }
 
+type ServiceExpose struct {
+	Enabled                  bool                                    `json:"enabled,omitempty"`
+	Type                     corev1.ServiceType                      `json:"type,omitempty"`
+	LoadBalancerSourceRanges []string                                `json:"loadBalancerSourceRanges,omitempty"`
+	Annotations              map[string]string                       `json:"annotations,omitempty"`
+	TrafficPolicy            corev1.ServiceExternalTrafficPolicyType `json:"trafficPolicy,omitempty"`
```

Comment:
You don't actually use this, overriding it with a hard-coded value based on Type.